custom/packaging/rockylinux-10.Dockerfile

```
@@ -0,0 +1,8 @@
+FROM rockylinux/rockylinux:10
+
+RUN yum update -y \
+  && yum install -y python3 python3-devel python3-pip openssl git rpmdevtools rpmlint \
+    systemd-units libxcrypt-compat git gnupg2 jq createrepo rpm-sign epel-release rustc cargo \
+  && yum install -y patchelf \
```

rpmlint comes from EPEL10, so it needs to be installed separately, after epel-release

```suggestion
  && yum install -y python3 python3-devel python3-pip openssl git rpmdevtools \
    systemd-units libxcrypt-compat git gnupg2 jq createrepo rpm-sign epel-release rustc cargo \
  && yum install -y patchelf rpmlint \
```

---

Thanks! Nice catch. Have applied and re-running pipelines.